agent/bench-scripts/pbench-fio

```diff
@@ -68,9 +68,9 @@ tool_label_pattern="fio-"
 tool_group="default"
 max_key_length=20
 primary_metric="readwrite_IOPS"
-histogram_interval_msec=$(getconf.py histogram-interval-msec pbench-fio)
```

Comment:
For compatibility, should we do something like:

```
# Fetch the new configuration value in seconds as a string.
histogram_interval_sec_str=$(getconf.py histogram-interval-sec pbench-fio)
if [ -z "$histogram_interval_sec_str" ] ;then
        # Fall back to old configuration value in milliseconds, and convert to seconds.
        histogram_interval_msec_str=$(getconf.py histogram-interval-msec pbench-fio)
        if [ -z "$histogram_interval_msec_str" ] ;then
                let histogram_interval_msec=10000
        else
                # Explicitly convert from string to a number
                let histogram_interval_msec=${histogram_interval_msec_str}*1
        fi
        let histogram_interval_sec=${histogram_interval_msec}/1000
else
        # Explicitly convert from string to a number
        let histogram_interval_sec=${histogram_interval_msec_str}*1
fi
```

Reply:
we can't really fall back without changing fio-histo-log-pctiles.py because its --time-quantum parameter is in units of seconds, so it doesn't make sense to use milliseconds anymore.  I thought we all agreed (or at least Andrew did) to go with seconds for this parameter because no one was interested in sub-second histogram logs.

Reply:
Is the proposal above to fall back to milliseconds or to seconds?  The above appears to propose the final result is in seconds, no?